### DISCUSSION TITLE
GPDR compatible logging

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1200] [Ideas]
At the moment, as soon as you activate logging with setting info or debug, the ip addresses are all visible in full. It should give a GPDR option to censor the output of the IP adresses, either set them to 0.0.0.0 or something like 192.000.000.192. 
Of course this has to be adjusted after the banning manager function.

With this, you still have some information and can troubleshoot, without having the full IP information saved in your logs.

**Comment by oliverw** [comment]
Converting to issue.
